support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/ValidatePasswordResetTokenActionTests.java

```diff
@@ -31,6 +39,24 @@ void verifyInvalidTicket() throws Throwable {
         assertEquals(CasWebflowConstants.TRANSITION_ID_INVALID_PASSWORD_RESET_TOKEN, validatePasswordResetTokenAction.execute(context).getId());
     }
 
+    @Test
+    void verifyGoodTicket() throws Throwable {
+        ClientInfoHolder.setClientInfo(new ClientInfo());
```

Comment:
This should be something like this:

```java
        val request = new MockHttpServletRequest();
        request.setRemoteAddr("185.86.151.11");
        request.setLocalAddr("185.88.151.11");
        val clientInfo = ClientInfo.from(request);
        ClientInfoHolder.setClientInfo(clientInfo);
```

And should either be moved to a before-each, or you must reset it for every test. This is a static singleton and setting it only in one test has side effects, or could have.

Reply:
I was in the rush of making the test pass but yes, the setup should be done properly.

I will move the logic in a before-each method.